### DISCUSSION TITLE
How to solve it

### DISCUSSION THREAD

**Discussion by Tharun199** [github/india#869] [GitHub Constellation 🇮🇳 2022 Q&A]
![Screenshot_2022-07-25-11-05-26-32_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/109937143/180705841-9dece0ac-8aa1-4ba9-a29f-1f8ccbdd1e61.jpg)
How to solve it 😢

**Comment by FRTMentor** [comment]
Please enter a valid Emaild in the given FutureSkillPrime textbox, 2nd field in the top.

**Comment by RijoKsd** [comment]
Hello @Tharun199  contact with future skills email
